actions/snow.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import requests
+import json
+import pathlib
+import ruamel.yaml
+from typing import Dict, Text, Any
+
+logger = logging.getLogger(__name__)
+
+here = pathlib.Path(__file__).parent.absolute()
```

Comment:
Now that actions are in a sub-directory, this is looking for `snow_credentials.yml` in the `actions` directory. Should change to something like:

```
here = pathlib.Path().resolve()
```

Reply:
Came across this discussion -  https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573/6 - seems like it's kinda complicated which is actually going to do what we want?

Reply:
Both of these seem to do the same - when is it that it breaks? when running in docker?

subdir/here.py
```
import pathlib
here = pathlib.Path(__file__).parent.absolute()
print(here)
here = pathlib.Path(__file__).parent.resolve()
print(here)
```

```
$ python subdir/here.py
/Users/melinda/subdir
/Users/melinda/subdir
```

Reply:
Ah, just realized, you might mean that `snow_credentials` isn't in `actions` at all, but I moved it there

Reply:
@rgstephens are you still seeing an issue with this or is it working for you?